autopilot/autopilot.go

```diff
@@ -301,6 +301,10 @@ func (ap *Autopilot) Run() error {
 		select {
 		case <-ap.shutdownCtx.Done():
 			return nil
+		default:
+		}
+
+		select {
```

Comment:
This doesn't seem quite right

Reply:
We can either split this one up, or add an extra check at the top of the loop. I noticed that on shutdown we sometimes unnecessary have some autopilot logging because the cases here aren't evaluated in a deterministic order. So if a force scan was triggered or waiting for consensus to sync was interrupted right when we shut down we get some logs we can avoid on avoid.

Reply:
Yeah but in that case it's still the wrong way around. At first you'd check for all cases including shutdown and then you'd check again for shutdown to see if the shutdown case triggered or any of the other ones. Otherwise you are going to miss a shutdown that happens between checking for shutdown and then checking for triggers.

Reply:
Oh that's what you mean! Ok will do.